plugins/inputs/aliyuncms/aliyuncms.go

```diff
@@ -144,11 +154,19 @@ func (s *AliyunCMS) Init() error {
 	if err != nil {
 		return fmt.Errorf("failed to retrieve credential: %w", err)
 	}
-	s.client, err = cms.NewClientWithOptions("", sdk.NewConfig(), credential)
+	s.cmsClient, err = cms.NewClientWithOptions("", sdk.NewConfig(), credential)
 	if err != nil {
 		return fmt.Errorf("failed to create cms client: %w", err)
 	}
 
+	if s.Project == "acs_rds_dashboard" && lo.Contains(s.MetricServices, "rds") {
```

Comment:
Rather than using the external library, we can use `slices.Contains` no?

Reply:
You are right, since GO 1.21 there is the `slices.Contains` functionality for the non-experimental part. I have adapted the code. Thanks for the hint.